quipucords/scanner/openshift/entities.py

```diff
@@ -128,3 +155,27 @@ def __raise__(self):
 
 # update nested model references - this should always be the last thing to run
 _update_model_refs()
+
+
+@dataclass
+class Cpu:
```

Comment:
just a nitpick but I like to break CamelCase for class name conventions when the class represents an all uppercase acronym (like CPU). IoW, Cpu -> CPU.